recipes-benchmark/s-suite/s-suite_git.bb

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "Small collection of benchmarks for storage I/O"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b529aaa6a0c50f15d29f89609b5c22f3"
+
+SRCREV = "79698f645bfb28d0d966484ddad3a1efb562246d"
```

Comment:
when using SRCREV, you need to set PV, otherwise you will get 'random' non linear version number.

if this project does not make any release, you can use:
PV = "0.0+git${SRCPV}"